docs/source/concepts/bento.rst

```diff
@@ -59,100 +59,105 @@ with a ``bentofile.yaml`` build file. Here's an example from the
 
     Successfully built Bento(tag="iris_classifier:dpijemevl6nlhlg6")
 
-Similar to :doc:`saving a model </concepts/model>`, a unique version tag will be
-automatically generated for the newly created Bento.
-
-It is also possible to customize the Bento version string by specifying it in the
-:code:`--version` CLI argument. However this is generally not recommended. Only use it
-if your team has a very specific naming convention for deployable artifacts, e.g.:
+After built, each Bento is automatically tagged with a unique version. It is also possible to set a specific version using the ``--version`` option,
+but this is generally unnecessary. Only use it if your team has a very specific naming convention for deployable artifacts.
 
 .. code-block:: bash
 
-    » bentoml build --version 1.0.1
+    bentoml build --version 1.0.1
 
 .. note::
 
-    The Bento build process requires importing the ``bentoml.Service`` object
-    defined. This means, the build environment must have all its dependencies installed.
-    Support for building from a docker environment is on the roadmap, see :issue:`2495`.
+    Building a Bento involves importing the ``bentoml.Service`` object and its dependencies.
+    Make sure you have all the dependencies installed in the build environment.
+    Support for building from a Docker environment is on the roadmap; see :issue:`2495` for details.
 
-Advanced Project Structure
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Custom build context
+^^^^^^^^^^^^^^^^^^^^
 
-For projects that are part of a larger codebase and interacts with other local python
-modules; Or for projects containing multiple Bentos/Services, it may not be possible to
-put all service definition code and ``bentofile.yaml`` under the project's root
-directory.
+For projects that are part of a larger codebase and interact with other local Python
+modules or those containing multiple Bentos/Services, it might not be possible to
+put all Service definition code and ``bentofile.yaml`` in the project's root directory.
 
-BentoML allows placing the service definition file and bentofile anywhere in the project
-directory. In this case, the user needs to provide the ``build_ctx`` and
-``bentofile`` argument to the ``bentoml build`` CLI command.
+BentoML allows the placement of the Service definition and ``bentofile.yaml`` anywhere in the project directory.
+In such scenarios, specify the ``build_ctx`` and ``bentofile`` arguments when running the ``bentoml build`` command.
 
-build_ctx
-    Build context is your Python project's working directory. This is from where you
-    start the Python interpreter during development so that your local python modules
-    can be imported properly. Default to current directory where the
-    ``bentoml build`` takes place.
+* ``build_ctx``: The build context represents the working directory of your Python project. It's the location where the Python interpreter starts,
```

Comment:
```suggestion
* ``build_ctx``: The build context represents the working directory of your Python project. It will be prepended to the PYTHONPATH during build process,
```